infrastructure/modules/xc3/output.tf

```diff
@@ -32,6 +32,6 @@ output "private_ip" {
 }
 
 output "load_balancer_dns" {
- description = "DNS of the load balancer"
- value       = var.domain_name != "" ? var.domain_name : aws_lb.this.dns_name
+  description = "DNS of the load balancer"
```

Comment:
```suggestion
  description = "DNS of the XC3 Dashboard"
```

Reply:
Added in 44dba3bb1d9fa24fab026017c0f60e673cf9c1ef